### DISCUSSION TITLE
✅ Multiple terminals

### DISCUSSION THREAD

**Discussion by jmjos** [isontheline/pro.webssh.net#240] [Q&A] [answered]
Hi!

Thanks for developing webssh! Before purchasing, I have a short question: How can I use "Multiple terminals"? Does that feature mean, that I can have two or more terminals open simultaneously and use them side-by-side? I can't figure that out by using the app.

Thanks,
Moritz

**Comment by isontheline** [comment, marked as answer]
Hi @jmjos Moritz 👋

Thanks for using WebSSH!

You can use : https://webssh.net/documentation/help/howtos/launching-multiple-terminals/
And you don't need to purchase WebSSH to enable this feature : you could try to open an SFTP and an SSH for testing for example.

Currently I don't support side-by-side, but I will do in the next few months.

Have a nice day ☀️

**Reply by RavenSystem** [reply]
Hello @isontheline 

Is it possible to use iPad split-screen feature? Years ago, I used WebSSH Pro and WebSSH Essential (with Ads) with split-screen.

Many thanks!

**Reply by isontheline** [reply]
Hello @jmjos 

Split-screen with other apps : YES
Split-screen with multiple terminals : NO(T YET)

![image](https://user-images.githubusercontent.com/44212923/120082548-ea7abc80-c0c3-11eb-94d2-7cda8808fc1a.png)
